service/discovery/azure/service_fabric.go

```diff
@@ -80,6 +80,7 @@ func (d *azureServiceFabricDiscovery) discoverClusters() ([]voc.IsCloudResource,
 	}
 
 	var clusters []*armservicefabric.Cluster
+	// TODO(all): I think it would be better to use the listPager function.
```

Comment:
see comment above

Reply:
I think we can use the listPager function here. If  I see it right, there is a NewListAllPager and NewListPager method. 

I've changed the TODOs.